app/assets/javascripts/discourse/app/components/topic-map/topic-map-expanded.gjs

```diff
@@ -0,0 +1,130 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { action } from "@ember/object";
+import { htmlSafe } from "@ember/template";
+import DButton from "discourse/components/d-button";
+import TopicParticipants from "discourse/components/topic-map/topic-participants";
+import replaceEmoji from "discourse/helpers/replace-emoji";
+import i18n from "discourse-common/helpers/i18n";
+import I18n from "discourse-i18n";
+import and from "truth-helpers/helpers/and";
+import lt from "truth-helpers/helpers/lt";
+import not from "truth-helpers/helpers/not";
+
+export default class TopicMapExpanded extends Component {
+  @tracked allLinksShown = false;
+  truncatedLinksLimit = 5;
+
+  @action
+  showAllLinks() {
+    this.allLinksShown = true;
+  }
+
+  get participantsTitle() {
+    return htmlSafe(`<h3>${I18n.t("topic_map.participants_title")}</h3>`);
+  }
```

Comment:
Any reason this needs to be a getter? Feels like it might be cleaner to move the `<h3>` into the TopicParticipants template rather than manually handling strings of html?

Something like 

```hbs
{{#if @title}}
  <h3>{{@title}}</h3>
{{/if}}
```